web/core/store/issue/issue-details/attachment.store.ts

```
@@ -105,20 +126,51 @@ export class IssueAttachmentStore implements IIssueAttachmentStore {
+        file,
+        (progressEvent) => {
+          runInAction(() => {
+            const progressPercentage = Number(((progressEvent.progress ?? 0) * 100).toFixed(0));
+            set(this.attachmentsUploadStatusMap, [issueId, tempId, "progress"], progressPercentage);
+          });
+        }
+      );
+      const issueAttachmentsCount = this.getAttachmentsByIssueId(issueId)?.length ?? 0;
 
-    return response;
+      if (response && response.id) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)